webapp/channels/src/components/sidebar/invite_members_button.tsx

```diff
@@ -52,11 +38,11 @@ const InviteMembersButton = (props: Props): JSX.Element | null => {
 
     let buttonClass = 'SidebarChannelNavigator__inviteMembersLhsButton';
 
-    if (!props.touchedInviteMembersButton && Number(totalUserCount) <= Constants.USER_LIMIT) {
+    if (!props.touchedInviteMembersButton) {
```

Comment:
I guess the only new behaviour from this is that the button will be a bit more highlighted until you click it for the first time. Before that wouldn't happen if you had more than 10 users